addons/website/static/src/components/configurator/configurator.js

```diff
@@ -127,33 +158,35 @@ class DescriptionScreen extends Component {
             results = results.concat(relaxedMatches);
             labels = results.map((val) => val.label);
         }
-        results.forEach((r) => {
-            this.labelToId[r.label] = r.id;
-        });
+        if (!results.length) {
+            labels = [_t("No result found, broaden your search.")];
+            this.autocompleteHasResults = false;
+        } else {
+            results.forEach((r) => {
+                this.labelToId[r.label] = r.id;
+            });
+            this.autocompleteHasResults = true;
+        }
```

Comment:
```javascript
this.autocompleteHasResults = !!results.length;
if (this.autocompleteHasResults) {
    // ...
} else {
    // ...
}
```

Also, the whole code above this is building a `labels` array then you ignore it totally and reset it if there is no result... there is better to do around here...

Reply:
If there is no results then this labels array is empty so we do not build it to ignore it afterwards but anyway this code indeed needed to be improved